data_safe_haven/backend/backend.py

```diff
@@ -59,9 +57,8 @@ def create(self) -> None:
                 tags=self.tags,
             )
             if not storage_account.name:
-                raise DataSafeHavenAzureException(
-                    f"Storage account '{self.config.backend.storage_account_name}' was not created."
-                )
+                msg = f"Storage account '{self.config.backend.storage_account_name}' was not created."
+                raise DataSafeHavenAzureError(msg)
```

Comment:
I'm not sure I find this clearer. We create a variable that is only used on one line (the following one) rather than using brackets and a line break. Is it the indentation that's the issue here?

Reply:
[Keeps tracebacks easy (at least easier) to read](https://beta.ruff.rs/docs/rules/f-string-in-exception/)

That is a new one to me :smile:. Sounds worth keeping.